.github/ISSUE_TEMPLATE/feature-request.yaml

```diff
@@ -3,12 +3,46 @@ description: Request a new feature
 labels:
   - enhancement
 body:
+  - type: checkboxes
+    id: before-posting
+    attributes:
+      label: "This request respects the following points:"
+      description: All conditions are **required**.
+      options:
+        - label: This request is **not** already on [GitHub](https://github.com/jellyfin/jellyfin-androidtv/labels/enhancement) _(I've searched it)_.
+          required: true
+        - label: I agree to follow Jellyfin's [Code of Conduct](https://jellyfin.org/docs/general/community-standards.html#code-of-conduct).
+          required: true
+  - type: textarea
+    attributes:
+      label: Problem Description
+      description: Describe the issue you are experiencing or the gap you are trying to address.
+      placeholder: |
+        The Android TV Client does not support displaying lyrics, while the Jellyfin Web client does.
+        This creates an inconsistent user experience across platforms.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Proposed Solution
+      description: Provide a detailed description of what you would like to see implemented or changed.
+      placeholder: |
+        The Client could display lyrics as an overlay on the screensaver. Additionally, a lyrics section could be integrated into the mini-player.
+    validations:
+      required: true
   - type: textarea
-    id: description
     attributes:
-      label: Describe the feature you'd like
-      description: |
-        A clear and concise description of what you want to request.
-        You can also attach screenshots or screen recordings to help explain your request.
+      label: Alternatives Considered
+      description: Outline any other approaches you have thought about or explored to solve the problem.
+      placeholder: |
+        An alternative approach could involve adding a dedicated "Lyrics" button in the Android TV Client, similar to the Jellyfin Web client.
+        Users could press this button to open a full-screen lyrics view.
     validations:
       required: true
+  - type: textarea
+    attributes:
+      label: Additional Information
+      description: Include any relevant details, resources, or screenshots that might help in understanding or implementing the request.
+      placeholder: Add any additional context here.
+    validations:
+      required: false
```

Comment:
Maybe add the "Other sources" here too so related forum posts/Fider requests could be linked.